change/@fluentui-react-tree-0bbd4942-7582-456a-9fdd-9027eaa0af91.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31780&commitId=6fd85504167536e648a9310c99954b1d8f410bf4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>